pkg/apicapi/apicapi.go

```diff
@@ -642,10 +654,10 @@ func (conn *ApicConnection) getSubtreeDn(dn string, respClasses []string,
 		return
 	}
 	for _, obj := range apicresp.Imdata {
-		//conn.log.WithFields(logrus.Fields{
-		//	"dn":  obj.GetDn(),
-		//	"obj": obj,
-		//}).Debug("Object updated on APIC")
+		conn.log.WithFields(logrus.Fields{
+			"dn":  obj.GetDn(),
+			"obj": obj,
+		}).Debug("Object updated on APIC")
```

Comment:
I suspect this commented out because it was too verbose even at debug level. We might just need to watch out.